README.md

```diff
@@ -239,7 +241,7 @@ foreach (var model in models)
 
 #### [Retrieve model](https://beta.openai.com/docs/api-reference/models/retrieve)
 
-Retrieves a model instance, providing basic information about the model such as the owner and permissioning.
+Retrieves a model instance, providing basic information about the model such as the owner and permissions.
```

Comment:
some of these changes look sus. This should already be what's in the main branch.